### PR TITLE
feat: Added search with case reference (FPLA-1414)

### PR DESCRIPTION
**Pull request by mounikahmcts**
https://tools.hmcts.net/jira/browse/FPLA-1414

Screenshot 2020-03-23 at 08 14 36
Screenshot 2020-03-23 at 08 14 44
Screenshot 2020-03-23 at 08 15 16
Screenshot 2020-03-23 at 08 14 20

### CODE REVIEW

File: ccd-definition/SearchResultFields.json [hunk 1]
```diff
@@ -1,37 +1,44 @@
 [
+  {
```
Comment (swalker125) [inline ccd-definition/SearchResultFields.json]: we don't want this to appear in the search results, please remove

File: ccd-definition/WorkBasketResultFields/WorkBasketResultFields.json [hunk 2]
```diff
@@ -1,37 +1,44 @@
 [
+  {
```
Comment (swalker125) [inline ccd-definition/WorkBasketResultFields/WorkBasketResultFields.json]: we don't want this to appear in the search results please remove

File: e2e/tests/hmctsAdministersCaseAfterSubmission_test.js [hunk 3]
```diff
@@ -239,9 +239,20 @@ Scenario('HMCTS admin handles supplementary evidence', async (I, caseListPage, c
...
 
+Scenario('HMCTS admin search for a given caseId', async (I, caseListPage) => {
+  await I.navigateToCaseList();
+  await caseListPage.searchForCasesWithCaseRef('0000');
+  await I.dontSeeCaseInSearchResult(caseId);
+
+  await I.navigateToCaseList();
+  await caseListPage.searchForCasesWithCaseRef(caseId);
+  await I.seeCaseInSearchResult(caseId);
+});
```
Comment (swalker125) [inline e2e/tests/hmctsAdministersCaseAfterSubmission_test.js]: we said we would not need to e2e test because we trust this functionality works as expected

**Comment by swalker125** [comment]
please update PR name to follow structure `feat: what it does FPLA-xxx`

see https://www.conventionalcommits.org/en/v1.0.0/ for commit conventions

File: ccd-definition/SearchResultFields.json (hunk 1 above)
Comment (mounikahmcts) [reply ccd-definition/SearchResultFields.json]: Followed screenshots from divorce and added it in results as well. But it is not provided ticket description removed it from results now

**Comment by mounikahmcts** [comment]
> please update PR name to follow structure `feat: what it does FPLA-xxx`
> 
> see https://www.conventionalcommits.org/en/v1.0.0/ for commit conventions

Updated it. Thanks 👍

**Pull request merged**